src/org/javarosa/xform/parse/XFormParser.java

```diff
@@ -188,36 +191,30 @@ private static void initProcessingRules() {
                     // Attributes that are passed through to additionalAttributes but shouldn't lead to warnings.
                     // These are consistently used by clients but are expected in additionalAttributes for historical
                     // reasons.
-                    final List<String> passedThroughInputAtts = Collections.unmodifiableList(Arrays.asList(
-                        "rows",
-                        "query"
-                    ));
-                    p.parseControl((IFormElement) parent, e, Constants.CONTROL_INPUT,
-                        passedThroughInputAtts,
-                        passedThroughInputAtts
-                    );
+                    List<String> passedThroughInputAtts = unmodifiableList(asList("rows", "query"));
```

Comment:
I also agree with putting these two string literals side by side. I wonder what my threshold is for wanting them in a column (or columns).

Reply:
Set it to 999 :D

Reply:
Right! I forgot who I was talking to. 😀   Mr. “I probably have one of these:”

![dell-u3415w-ultrawide-curved-monitor-review-front-full-2-720x720](https://user-images.githubusercontent.com/55099/37945563-b0fed8fe-3135-11e8-904e-0ec3c78b9a8d.jpg)

Reply:
I'm happier since I discovered that shift+mousewheel scrolls horizontally :)